### PR TITLE
KFF-3974 Travis tests updates and improvements

### PR DESCRIPTION
**Pull request by kevinfodness**
* Preserve WP 4.9 support by not calling `register_post_meta` if not exists.
* Fix issues arising from older versions of PHP and WordPress.
* Fix bug with `phpunit` path that was resulting in the wrong version of `phpunit` getting called.
* Adopt the Alley coding standard for phpcs, run phpcs through composer, fix found issues.

### CODE REVIEW

File: assets.php [hunk 1]
```diff
@@ -39,7 +35,7 @@ function action_enqueue_block_editor_assets() {
  *
  * @return array An array of dependencies for this asset.
```
Comment (jomurgel) [inline assets.php:36]: I can't tell based on the diff, perhaps this is missing a @param comment?

File: assets.php [hunk 2]
```diff
@@ -68,21 +63,43 @@ function get_asset_dependencies( string $asset ) : array {
  *
  * @return string The asset's hash.
  */
```
Comment (jomurgel) [inline assets.php:65]: This also doesn't have a @param comment. Needed?

**Review by jomurgel — APPROVED** [review]
Maybe a couple missing param comments, otherwise 👍 b

File: assets.php (hunk 1 above)
Comment (kevinfodness) [reply assets.php:36]: Nope, there aren't any params on this function.

File: assets.php (hunk 2 above)
Comment (kevinfodness) [reply assets.php:65]: This has a param tag - it's a line above this.

**Pull request merged**